packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1260,7 +1268,9 @@ The base user profile is based on the [System for Cross-Domain Identity Manageme
 | managerId         | `id` of a user's manager                                                                                                     | String   | TRUE     | FALSE  | FALSE    |           |           |                                                                                                                   |
 | manager           | displayName of the user's manager                                                                                            | String   | TRUE     | FALSE  | FALSE    |           |           |                                                                                                                   |
 
-> Note: A locale value is a concatenation of the ISO 639-1 two letter language code, an underscore, and the ISO 3166-1 2 letter country code; e.g., 'en_US' specifies the language English and country US.
+>Note: A locale value is a concatenation of the ISO 639-1 two letter language code, an underscore, and the ISO 3166-1 2 letter country code; e.g., 'en_US' specifies the language English and country US.
```

Comment:
As long as we're updating these, can we make **Note:** bold here and throughout?

Reply:
This feels like Yet Another case of a style change that ought to be applied throughout the API docs instead of just in this file.  I checked all other docs/reference/api/*/index.md files and found more than 50 lines starting with `>Note:` (or variants with a space after the `>`) but only 1 that used `**Note:**` (and it does not include the leading `>`).  So again, if you feel this formatting should be cleaned up, I think you should file a separate JIRA for that and address it simultaneously across all the API docs.

For this PR, I am going to invoke the default programming style rule, which is to follow the predominant existing style.

Reply:
https://github.com/okta/okta-developer-docs/pull/679/files

Reply:
_Fine._  I'll embolden the Notes in my PR, and have opened OKTA-250196 to have someone (hopefully not me) address all the other Notes in the docs/reference/api portion of the site.